src/styles/product-card.scss

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: $transition;
+
+  &--secondary {
+    cursor: pointer;
+    box-sizing: border-box;
```

Comment:
it's better to apply to each element on the page (`*` selector) and remove everywhere else